recipes/pysurfer/meta.yaml

```diff
@@ -0,0 +1,63 @@
+# To update, change version number and sha256 -- you can get it for a file with:
+#  `openssl sha256 <file name>`.
+# You may need the openssl package, available on conda-forge
+#  `conda install openssl -c conda-forge``
+{% set version = "0.7" %}
+{% set sha256 = "c003fa83d7214014e55ce5b831d622acabe935b563810441b991fa1a47ec9fae" %}
+
+package:
+  name: pysurfer
+  version: {{ version }}
+
+source:
+  fn: pysurfer-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/p/pysurfer/pysurfer-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py>=30 or linux]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - imageio
+    - matplotlib
+    - mayavi
+    - nibabel
+    - numpy
+    - scipy
+  run:
+    - python
+    - imageio
+    - matplotlib
+    - mayavi
+    - nibabel
+    - numpy
+    - scipy
+
+test:
+  commands:
+    - pysurfer --help                                              # [not linux]
+    - DISPLAY=localhost:1.0 xvfb-run -a bash -c 'pysurfer --help'  # [linux]
+
+about:
+  home: http://pysurfer.github.com
+  license: BSD-3-clause
+  license_family: BSD
```

Comment:
The license file is in the repository, but it's not included in the source distribution. It could probably be added in future versions.